contracts/ROSCA.sol

```diff
@@ -85,7 +85,7 @@ contract ROSCA {
   bool public endOfROSCA = false;
   bool public forepersonSurplusCollected = false;
   // A discount is the difference between a winning bid and the pot value. totalDiscounts is the amount
-  // of discounts accumulated so far, divided by the number of ROSCA participants.
+  // of discounts accumulated so far
```

Comment:
The calculation and probably the meaning of `totalDiscounts` is are changed after this pull request. 

Do we want to change the variable name too? 

If `totalDiscounts` name is still the intention, then the name is totally inappropriate in the past.

Reply:
there is not a better name for this variable IMHO.